draft-ietf-quic-multipath.md

```diff
@@ -319,7 +319,11 @@ Path ID 0.
 After the handshake concluded if negotiation of multipath support succeeded,
 endpoints SHOULD use PATH_ACK frames instead of ACK frames,
 also for acknowledging so far unacknowledged 0-RTT packets, using
-Path ID 0. Similarly after a successful handshake, endpoints SHOULD also use
+Path ID 0. Further more, endpoints SHOULD use PATH_ACK frames in all 1-RTT packets after
+the successful negotiation for multipath. If endpoints receive ACK frame
+after a succesful negotiation resulting from packet out-of-order arrival,
+endpoints could consider it as used for initial PATH ID 0 without causing connection error.
+Similarly after a successful handshake, endpoints SHOULD also use
 the PATH_NEW_CONNECTION_ID frame to provide new connection IDs for Path ID 0 and,
 respectively, the PATH_RETIRE_CONNECTION_ID frame to retire connection IDs for
 Path ID 0.
```

Comment:
I think I agree that the added text "Further more, endpoints SHOULD use PATH_ACK frames in all 1-RTT packets after
the successful negotiation for multipath. If endpoints receive ACK frame after a succesful negotiation resulting from packet out-of-order arrival, endpoints could consider it as used for initial PATH ID 0 without causing connection error." is not what we want. I would replace it by something like:

"However, even after successful negotiation, endpoints MAY use ACK frames  to acknowledge 0RTT packets or 1RTT packets received with PathID=0."

Stating the MAY implicitly states that arrival of such packets should not be treated as a connection error.

Reply:
I'm fine to add this sentence to make it explicit that those packets arrive. However, I'm not sure about the use of normative language here. Because this is not to say, you are aloof to send that (that's already included in the SHOULD above). It's to say, please be aware that they may arrive and you have to handle them. Maybe like this:

> Still, even after successful negotiation, any ACK frames that acknowledge 0-RTT packets or 1-RTT packets for path ID 0 need to be processed correctly.